modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,167 @@
+
+.. Complete the required fields in *Form View* or click *YAML view* for full control.
+
+.. When finished, click *Create*. The new result is displayed.
+. To display a continuous list of available devices on a node:
+
+.. Log in to the {product-title} web console.
+
+.. Navigate to *Compute* -> *Nodes*.
+
+.. Click the node name that you want to open. The "Node Details" page is displayed.
+
+.. Select the *Disks* tab to display the list of the selected devices.
++
+The device list updates continuously as local disks are added or removed. You can filter the devices by name, status, type, model, capacity, and mode.
++
+[NOTE]
+====
+Results are deleted after they are removed from the node. Symlinks must be manually removed.
+====
+
+. To automatically provision local volumes for the discovered devices from the web console:
+
+.. Navigate to *Operators* -> *Installed Operators* and select *Local Storage* from the list of Operators.
+
+.. Select *Local Volume Set* -> *Create Local Volume Set*.
+
+.. Enter a volume set name and a storage class name.
+
+.. Choose *All nodes* or *Select nodes* to apply filters accordingly.
+
+.. Select the disk type, mode, size, and limit you want to apply to the local volume set, and click *Create*.
++
+A message displays after several minutes, indicating that the "Operator reconciled successfully."
+
+[start=3]
+. Alternately, to provision local volumes for the discovered devices from the CLI:
+
+.. Create an object YAML file to define the local volume set, such as `local-volume-set.yaml`, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: local.storage.openshift.io/v1alpha1
+kind: LocalVolumeSet
+metadata:
+  name: example-autodetect
+spec:
+  nodeSelector:
+    nodeSelectorTerms:
+      - matchExpressions:
+          - key: kubernetes.io/hostname
+            operator: In
+            values:
+              - worker-0
+              - worker-1
+  storageClassName: example-storageclass <1>
+  volumeMode: Filesystem
+  fstype: ext4
+  maxDeviceCount: 10
+  deviceInclusionSpec:
+    deviceTypes: <2>
+      - disk
+      - part
+    deviceMechanicalProperty:
+      - NonRotational
+    minSize: 10G
+    maxSize: 100G
+    models:
+      - SAMSUNG
+      - Crucial_CT525MX3
+    vendors:
+      - ATA
+      - ST2000LM
+status:
+  conditions:
+  - lastTransitionTime: "2020-10-12T16:48:47Z"
+    message: 'DiskMaker: Available, LocalProvisioner: Available'
+    status: "True"
+    type: DaemonSetsAvailable
+  - lastTransitionTime: "2020-10-12T16:48:47Z"
+    message: Operator reconciled successfully.
+    status: "True"
+    type: Available
+  observedGeneration: 1
+  totalProvisionedDeviceCount: 0
+----
++
+<1> Determines the storage class created for persistent volumes that are provisioned from discovered devices.
++
+<2> When using the local volume set feature, Local Storage Operator does not support the use of logical volume management (LVM) devices.
```

Comment:
Should this be ', the Local Storage Operator...'?

Reply:
I always wonder and I'm a flip-flopper. I know we dropped the article when writing`OperatorHub`, but not sure about the Operator names themselves. @adellape, do you know?